<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1096</width>
    <height>866</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="displayGraphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>720</width>
      <height>720</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="leftButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>180</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Left</string>
    </property>
    <property name="shortcut">
     <string>Left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="centerButton">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>180</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Button</string>
    </property>
    <property name="shortcut">
     <string>Space</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rightButton">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>180</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Right</string>
    </property>
    <property name="shortcut">
     <string>Right</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="switchUp">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>70</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>switch up</string>
    </property>
    <property name="shortcut">
     <string>Q</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">switchGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="switchNeutral">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>90</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>neutral</string>
    </property>
    <property name="shortcut">
     <string>A</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">switchGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="switchDown">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>110</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>switch down</string>
    </property>
    <property name="shortcut">
     <string>Z</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">switchGroup</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1096</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="switchGroup"/>
 </buttongroups>
</ui>
